_fellows/abrar-qureshi.md

```diff
@@ -10,6 +10,8 @@ region: east
 bio_short: A technologist and a believer in continuous re-imagination and simplification.
 tags: ['east', '2021']
 skills: ['frontend development', 'data', 'backend development']
+linkedin: test
+twitter: test
```

Comment:
This is an example of how to add the LinkedIn and Twitter usernames. We'll want to remove this prior to merging this PR.